### PR TITLE
Feature/smugglers satchel

### PR DESCRIPTION
**Pull request by Ermucat**
## About the PR
Added the smugglers satchel, a new syndiecate item that can fit under floor tiles

## Why / Balance
New way to hide loot, it can also be seen with tray so its not to powerful

## Technical details
=Added entities to list

## Media
See below I guess?

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase

https://github.com/user-attachments/assets/f07adfc9-c5be-4e4b-98e9-8b1e63ac01eb

.

## Breaking changes

**Changelog**
:cl:
- add: Added Smuggler's Satchel, a stealthy syndicate item

### CODE REVIEW

**Comment by Unisol** [comment]
Should it be considered contraband? If it's supposed to be clandestine backpack that isn't illegal by itself, I'd suggest making sprite the same as base satchel; otherwise, it's trivial to metagame.

Alternatively, give it a non-uplink way of obtaining (such as adding it to maints locker spawn pool); this way you won't be immediately valid if you have this.

**Comment by Ermucat** [comment]
Oh shoot your right let me figure out how to do that real quick

**Comment by RotEmpress** [comment]
Out of curiousity, Ermu. what's your thoughts on allowing Smugglers Satchel to be spawned around the station underneath tiles? I feel like if we're basing it off of TG why not allow that to be a possibility? Or did you just want the feature in first, more things later?

**Comment by Ermucat** [comment]
I know there is an upstream Pr that adds that, I see no problem in having them spawn randomly, other than my version being syndie contraband

File: Resources/Locale/en-US/store/uplink-catalog.ftl [hunk 1]
```diff
@@ -451,3 +451,6 @@ uplink-combat-bakery-desc = A kit of clandestine baked weapons. Contains a bague
 
 uplink-business-card-name = Syndicate Business Card
 uplink-business-card-desc = A business card that you can give to someone to demonstrate your involvement in the syndicate or leave at the crime scene in order to make fun of the detective. You can buy no more than three of them.
+
+uplink-smuggler-satchel-name = Smugglers Satchel
+uplink-smuggler-satchel-desc = A satchel that can fit under station tiles.
```
Comment (FluffMe) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:456]: These loc strings should be introduced in `Resources/Locale/en-US/_Harmony/store/uplink-catalog.ftl` instead.

File: Resources/Prototypes/Entities/Clothing/Back/satchel.yml [hunk 2]
```diff
@@ -178,3 +178,22 @@
...
+    grid:
+    - 0,0,1,3
+    - 3,0,6,3
+    - 8,0,9,3
+  - type: Sprite
+    sprite: Clothing/Back/Satchels/smuggler.rsi
+    visible: false
+  - type: SubFloorHide
+  - type: Anchorable
+  - type: Appearance
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Back/satchel.yml:199]: This entity should be introduced in `Resources/Prototypes/_Harmony/Entities/Clothing/Back/satchel.yml` instead.

File: Resources/Textures/Clothing/Back/Satchels/smuggler.rsi/meta.json
Comment (FluffMe) [inline Resources/Textures/Clothing/Back/Satchels/smuggler.rsi/meta.json:1]: This RSI and all sprites in it should be relocated to `Resources/Textures/_Harmony/Clothing/Back/Satchels/smuggler.rsi`

**Review by FluffMe — CHANGES_REQUESTED** [review]

**Comment by FluffMe** [comment]
The changelog entry in the PR request seems to be commented out. Please adjust the description until it's properly showing.

File: Resources/Prototypes/Entities/Clothing/Back/satchel.yml (hunk 2 above)
Comment (FluffMe) [reply Resources/Prototypes/Entities/Clothing/Back/satchel.yml:199]: Additionally, `name` should be lowercase to match every other item name in the game.

File: Resources/Locale/en-US/store/uplink-catalog.ftl (hunk 1 above)
Comment (FluffMe) [reply Resources/Locale/en-US/store/uplink-catalog.ftl:456]: Additionally, `
uplink-smuggler-satchel-name` should be lowercase to match every other item name in the game.

**Comment by FluffMe** [comment]
Test fail is real:

```
   Failed UninitializedSaveTest [11 s]
  Error Message:
   Prototype ClothingSatchelSmugglers gains a component on spawn: CollideOnAnchor
```

**Review by FluffMe — DISMISSED** [review]

**Review by DieselMohawk — APPROVED** [review]

**Review by FluffMe — APPROVED** [review]

**Review by DieselMohawk — APPROVED** [review]

**Pull request merged**